src/java.base/share/classes/java/io/ObjectOutputStream.java

```diff
@@ -656,10 +656,11 @@ protected void writeStreamHeader() throws IOException {
      * stream.  Subclasses of ObjectOutputStream may override this method to
      * customize the way in which class descriptors are written to the
      * serialization stream.  The corresponding method in ObjectInputStream,
-     * {@code readClassDescriptor}, should then be overridden to
+     * {@link ObjectInputStream#readClassDescriptor readClassDescriptor}, should then be overridden to
```

Comment:
This makes the line lengths in this paragraph a bit inconsistent, maybe it can be re-formatted so that this one line doesn't stick out so much.